### PR TITLE
adding a way to toggle the welcome message

### PR DESCRIPTION
**Pull request by NilsFedrigo**
please message me for any other info!

### CODE REVIEW

File: ilugd.py [hunk 1]
```diff
@@ -6,19 +6,18 @@
...
     bot.sendMessage(*args, **kwargs)
-
 logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',level=logging.DEBUG)
 
 config = configparser.ConfigParser()
 config.read('bot.ini')
 
 
-updater = Updater(os.environ['token']) # we should use env variable !!
+updater = Updater(token = '630074149:AAE5G411hgWo33tejBsHP6qxFc6eSO7S-m0') # we should use env variable !! <- I don't (^^)
```
Comment (hellozee) [inline ilugd.py:20]: One must not push his token to a public repository, also this bot already has one, which is set as the environment variable, please revert the line back

**Review by hellozee — CHANGES_REQUESTED** [review]
Remove the files related to your IDE or add them to the .gitignore, also remove the unnecessary code, instead of commenting them, right now it is pretty hard to follow your code.
Thanks for the patch

File: ilugd.py (hunk 1 above)
Comment (xeon-zolt) [reply ilugd.py:20]: also .idea should not be pushed it should be in .gitignore

File: .idea/inspectionProfiles/Project_Default.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<component name="InspectionProjectProfileManager">
```
Comment (xeon-zolt) [inline .idea/inspectionProfiles/Project_Default.xml:1]: remove

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (xeon-zolt) [inline .idea/misc.xml:1]: remove

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
```
Comment (xeon-zolt) [inline .idea/modules.xml:4]: remove

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (xeon-zolt) [inline .idea/vcs.xml:1]: remove

File: .idea/workspace.xml [hunk 6]
```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (xeon-zolt) [inline .idea/workspace.xml:1]: remove

**Review by xeon-zolt — CHANGES_REQUESTED** [review]
